### PR TITLE
MEN-2427: Document mender setup changes on deb package

### PR DESCRIPTION
**Pull request by lluiscampos**
This will go together with https://github.com/mendersoftware/mender-dist-packages/pull/16 and https://github.com/mendersoftware/mender/pull/443

```
commit 85a50821236e8699a4005358a496bc9eae6021b2
Author: Lluis Campos <lluis.campos@northern.tech>
Date:   Wed Oct 30 17:00:36 2019 +0100

    Document changes in deb package installation
    
    Now a single configuration step is required, which can be done
    interactively or unattended; so write the instructions for both cases.
    
    Changelog: None
    
    Signed-off-by: Lluis Campos <lluis.campos@northern.tech>
```

### CODE REVIEW

File: 05.Client-configuration/06.Installing/docs.md [hunk 1]
```diff
@@ -27,82 +28,46 @@ Download the package:
...
 ```bash
-sudo systemctl enable mender && sudo systemctl restart mender
+DEVICE_TYPE="<INSERT YOUR DEVICE TYPE>"
+TENANT_TOKEN="<INSERT YOUR TOKEN FROM https://hosted.mender.io/ui/#/settings/my-organization>"
+wget https://d1b0l86ne08fsf.cloudfront.net/master/dist-packages/debian/armhf/mender-client_master-1_armhf.deb
+sudo DEBIAN_FRONTEND=noninteractive dpkg -i mender-client_master-1_armhf.deb
+sudo mender setup \
+            --device-type $DEVICE_TYPE \
+            --demo \
+            --mender-professional \
```
Comment (mzedel) [inline 05.Client-configuration/06.Installing/docs.md]: so this equals setting the backend to Mender-professional/ Hosted Mender as backend if I understood the explanation correctly - just to clarify it for diverging cases: how would the command look like for other backends (as in other IPs, URLs and certificates required)

Comment (lluiscampos) [reply 05.Client-configuration/06.Installing/docs.md]: I updated the documentation presenting three use cases: Mender Professional, demo server, and Mender Enterprise server.

I think it should now be clearer for you and the reader.

**Comment by lluiscampos** [comment]
Related to the last comments above; discussed in person with @alfrunes,

He will rework a bit the state machine to resolve these two cases. We think this is a low effort change and will make it a lot more intuitive. After new code lands in `mender` repo, this docs PR will be reworked.

**Comment by lluiscampos** [comment]
Reworked following latest changes in `mender setup` command. @mzedel please have a look

**Pull request merged**